data/transit/route/bus.json

```
@@ -13379,6 +13379,7 @@
       "locationSet": {"include": ["th"]},
       "tags": {
         "network": "smart bus",
+        "network:wikidata": "Q116101493"
```

```suggestion
        "network:wikidata": "Q116101493",
```

Welcome and thank you for contribute to NSI. However, you need to add a comma to make it a valid json file. For more guide you can see https://github.com/osmlab/name-suggestion-index/wiki